src/hooks/pools/graphql/Pool.graphql

```diff
@@ -28,6 +28,7 @@ type LBPAssetWeights {
 }
 
 type Fee {
+  id: String
```

Comment:
This is quite counter productive since the id field wont be enforced at the resolver on a type level, instead you can use interfaces to have a fee with and without id, such as:

```
interface Fee {
  numerator: String!
  denominator: String!
}

type PoolFee implements Fee {
  numerator: String!
  denominator: String!
}

type ConstantFee implements Fee {
  id: String!,
  numerator: String!,
  denominator: String!
}
```

Reply:
Yes,  I see.  Without enforcement at the type/schema level,  the resolver could be changed more easily to stop returning the `id` and then our caching is not as robust.

I'll fix that

Reply:
@maht0rz just pushed fix for 👀